modules/merge_waiter/main.tf

```diff
@@ -22,7 +22,7 @@ resource "aws_lambda_function" "merge_waiter" {
 
 # IAM
 resource "aws_iam_role" "merge_waiter" {
-  name = "lambda-role-${var.app_name}-${var.env_type}-merge-waiter"
+  name = "${var.app_name}_${var.env_type}-merge-waiter"
```

Comment:
"lambda-role" is added to name as per coding standards. Any specific reason for removing it ?

Reply:
will fix, removed as reult of conflict